browser/ui/webui/brave_wallet/market/market_ui.cc

```diff
@@ -49,8 +54,8 @@ UntrustedMarketUI::UntrustedMarketUI(content::WebUI* web_ui)
       std::string("style-src 'self' 'unsafe-inline';"));
   untrusted_source->OverrideContentSecurityPolicy(
       network::mojom::CSPDirectiveName::StyleSrc,
-      std::string(
-          "style-src 'self' 'unsafe-inline' chrome-untrusted://resources;"));
+      std::string("style-src 'self' 'unsafe-inline' "
+                  "chrome-untrusted://resources chrome-untrusted://theme;"));
   untrusted_source->OverrideContentSecurityPolicy(
```

Comment:
Could you also please remove StyleSrc override above

Reply:
wow didn't even notice that!

Reply:
also `chrome-untrusted://theme` would go to android, not sure if we want it nor if it causes problems